### PR TITLE
Feature: Industry Page

### LINKED ISSUE #7: Phase 2: Industry page
CardTech Inc Industry Page part 1

- [x] Contains a list of Industry Categories

CardTech Inc Industry Page part 2

- [x] Categories will expand as a drop down menu will links

CardTech Inc Industry Page part 3

- [ ] Data specific to the link will be outputted to the user

For example

- [ ] pictures and data related to research and innovation

- [ ] locations of research centers

- [ ] types of research being conducted at a specific location

- [ ] location and research data will be pulled from the database

- [ ] for clarity purposes a spreadsheet output of the data would be ideal 

- [ ] Needs additional page for the working links on the page

### PR DESCRIPTION
**Pull request by diagnose**
This resolves #7.

### CODE REVIEW

**Review by JCDILL03 — APPROVED** [review]
The accordion style on that page looks great. No issues from me.

File: CardTech.ResearchSite/CardTech.ResearchSite/Pages/Industry/Centers.cshtml [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+                </button>
+            </h2>
+        </div>
+        <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#centersAccordion">
+            <div class="card-body">
+                <div class="list-group list-group-flush">
+                    With 10+ affiliated faculty from engineering, medicine and dentistry the AMIST supports and promotes the broad range of additive manufacturing (AM) research activities at the University of Louisville. Current faculty research is funded by industry and multiple federal agencies, including DoD (Navy, Air Force and Army), NASA and NSF.
+                    <br />
+                    <br />
+                    Additive Manufacturing Institute of Science & Technology
```
Comment (diagnose) [inline CardTech.ResearchSite/CardTech.ResearchSite/Pages/Industry/Centers.cshtml]: Should put this and other addresses in an `<address>` block

File: CardTech.ResearchSite/CardTech.ResearchSite/Pages/Industry/Centers.cshtml [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+            <button class="btn btn-link btn-block text-left collapsed" type="button" data-toggle="collapse" data-target="#collapseTen" aria-expanded="false" aria-controls="collapseTen">
+                CTSI-CTSA: Clinical and Translational Sciences Institute
+            </button>
+        </h2>
+    </div>
+    <div id="collapseTen" class="collapse" aria-labelledby="headingTen" data-parent="#centersAccordion">
+        <div class="card-body">
+            <div class="list-group list-group-flush">
+                The UofL and its Health Partners serve a highly diverse patient population, including many underrepresented and medical underserved groups. Our overall goals are to develop a collaborative clinical and translational research infrastructure across Kentucky and with partnering IDeA CTRs (Clinical and Translational Resources) and CTSAs (Clinical and Translational Science Awards) that will positively impact the health outcomes; and to develop an interactive professional development program that will create a pipeline for the next generation of clinical and translational investigators. This us achieved these goals through four aims:
+                <br />
```
Comment (diagnose) [inline CardTech.ResearchSite/CardTech.ResearchSite/Pages/Industry/Centers.cshtml:315]: Should probably move this and other lists into an `<ol>` block

**Review by diagnose — COMMENTED** [review]
`<br />` is a little ugly but the overall structure is great. I'd approve this just to get the functionality on the site tonight and we can clean up the code later. 👍🏼

**Pull request merged**